tst/EnergyPlus/unit/ChillerExhaustAbsorption.unit.cc

```diff
@@ -1155,3 +1156,280 @@ TEST_F(EnergyPlusFixture, ExhAbsorption_GetInput_Multiple_Objects_Test)
+        "    9.111418E-09,            !- Coefficient5 y**2                                                             ",
+        "    8.797885E-07,            !- Coefficient6 x*y                                                              ",
+        "    -17.8,                   !- Minimum Value of x                                                            ",
+        "    50.0,                    !- Maximum Value of x                                                            ",
+        "    0.0,                     !- Minimum Value of y                                                            ",
+        "    3050.,                   !- Maximum Value of y                                                            ",
+        "    ,                        !- Minimum Curve Output                                                          ",
+        "    ,                        !- Maximum Curve Output                                                          ",
+        "    Temperature,             !- Input Unit Type for X                                                         ",
+        "    Distance,                !- Input Unit Type for Y                                                         ",
+        "    Dimensionless;           !- Output Unit Type                                                              ",
+
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    compare_err_stream("");
+
+    GetExhaustAbsorberInput(*state);
+
+    auto &thisChillerHeater = state->dataChillerExhaustAbsorption->ExhaustAbsorber(1);
+
+    Real64 loadinput = -5000.0;
+    bool runflaginput = true;
+
+    thisChillerHeater.CoolingLoad = 100000.0;
+    thisChillerHeater.CoolPartLoadRatio = 1.0;
+    state->dataPlnt->TotNumLoops = 1;
+    state->dataPlnt->PlantLoop.allocate(state->dataPlnt->TotNumLoops);
+
+    thisChillerHeater.CWPlantLoc.loopNum = 1;
+    thisChillerHeater.CWPlantLoc.loopSideNum = DataPlant::LoopSideLocation::Demand;
+
+    state->dataPlnt->PlantLoop(1).FluidName = "WATER";
+    state->dataPlnt->PlantLoop(1).FluidIndex = 1;
+    state->dataPlnt->PlantLoop(1).LoopDemandCalcScheme = DataPlant::LoopDemandCalcScheme::SingleSetPoint;
+    state->dataPlnt->PlantLoop(1).LoopSide(DataPlant::LoopSideLocation::Demand).FlowLock = DataPlant::FlowLock::Locked;
+    state->dataLoopNodes->Node(3).Temp = 60.0;
+    state->dataLoopNodes->Node(3).MassFlowRate = 0.5;
+    state->dataLoopNodes->Node(4).TempSetPoint = 70.0;
+    state->dataLoopNodes->Node(7).Temp = 350.0;
+    state->dataLoopNodes->Node(7).MassFlowRate = 0.5;
+
+    thisChillerHeater.InCoolingMode = true;
+    thisChillerHeater.isWaterCooled = true;
+
+    thisChillerHeater.calcChiller(*state, loadinput);
+
+    EXPECT_EQ(state->dataErrTracking->TotalSevereErrors, 3);
+    EXPECT_EQ(state->dataErrTracking->LastSevereError,
+              "CalcExhaustAbsorberChillerModel: Condenser flow = 0, for Exhaust Absorber Chiller=EXH CHILLER");
```

Comment:
This line passes with the current develop branch code, was that the expected behavior?

Reply:
Yes, it is part of the expected behavior (and also required) for testing these error messages. I am also considering adding one check for "at end" part that you suggested.